### PR TITLE
GEOMESA-1743 Updating CCRi copyright to 2017

### PR DESCRIPTION
**Pull request by elahrvivaz**
Signed-off-by: Emilio Lahr-Vivaz <elahrvivaz@ccri.com>

### CODE REVIEW

File: build/copyright/ccri-dstl-copyright.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Copyright (c) ${project.inceptionYear}-2016 ${owner}
-Portions Crown Copyright (c) 2016 Dstl
+Copyright (c) ${project.inceptionYear}-2017 ${owner}
+Portions Crown Copyright (c) 2017 Dstl
```
Comment (aheyne) [inline build/copyright/ccri-dstl-copyright.txt]: Do we want to update copyrights for other groups?

Comment (elahrvivaz) [inline build/copyright/ccri-dstl-copyright.txt]: i didn't actually update them - just renamed some of the files

File: build/copyright/ccri-dstl-copyright-2017.txt [hunk 2]
```diff
@@ -1,6 +0,0 @@
-Copyright (c) ${project.inceptionYear}-2016 ${owner}
-Portions Crown Copyright (c) 2017 Dstl
```
Comment (aheyne) [inline build/copyright/ccri-dstl-copyright-2017.txt]: This doesn't make sense. If dstl is 2017 then why's it 2016 in the new build/copyright/ccri-dstl-copyright-2016.txt ?

Comment (elahrvivaz) [inline build/copyright/ccri-dstl-copyright-2017.txt]: they have some files marked 2017 and some marked 2016

Comment (elahrvivaz) [inline build/copyright/ccri-dstl-copyright-2017.txt]: i think the new files are right, it's just the diff view isn't very good... here's what we have after this PR:
`ccri` -> default, ccri, latest year
`ccri-dstl-copyright` -> ccri latest year, dstl 2017
`ccri-dstl-copyright-2016` -> ccri latest year, dstl 2016
`dstl-copyright-2016` -> dstl 2016
`dstl-copyright-2017` -> dstl 2017
`ibm` -> ibm 2017
`ibm-ccri` -> ibm 2017, ccri latest year

Comment (elahrvivaz) [inline build/copyright/ccri-dstl-copyright-2017.txt]: oops, there were static `2017`s in the dstl files, replaced those with `${year}`

**Pull request merged**